src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/ICodeFixHandler.cs

```diff
@@ -21,5 +21,7 @@ public interface ICodeFixHandler
             Engine engine,
             CheckResult checkResult,
             CancellationToken cancel);
+
+        void OnCodeActionApplied(CodeAction codeAction);
```

Comment:
>OnCodeActionApplied [](http://example.com/codeflow?start=13&length=19)

We should note that the handler receiving this callback may not be the same object instance that make the suggestion. (or may not even be on the same server)

Reply:
I don't think receiving server instance matters, as long as it knows how to handle callback (unless it has stateful info dependency)